src/Header/Header.stories.tsx

```diff
@@ -17,21 +17,21 @@ const Template: Story = args => <Header {...args} />;
 
 export const LogoOnly = Template.bind({});
 LogoOnly.args = {
-  children: <EdozoLogo appName="Unicorns" />,
+  children: <EdozoLogo2 appName="Unicorns" />,
 };
 
-export const WithMenu = Template.bind({});
-WithMenu.args = {
+export const MapsWithMenu = Template.bind({});
+MapsWithMenu.args = {
   children: (
     <Fragment>
-      <EdozoLogo appName="Unicorns" />
+      <EdozoLogo2 appName="maps" />
       <Menu>
-        <Menu.ItemWrapper>
+        <Menu.ItemWrapper appName="maps">
```

Comment:
On this I get a type error: `Type '{ children: Element; appName: string; }' is not assignable to type 'IntrinsicAttributes & { children?: ReactNode; }'.
  Property 'appName' does not exist on type 'IntrinsicAttributes & { children?: ReactNode; }'.ts(2322)`

Reply:
In Menu.tsx line 9 should be changed from `ItemWrapper: React.FC;` to `ItemWrapper: React.FC<MenuItemProps>;`

Reply:
Thank you, I've added it :)